README.md

```diff
@@ -51,6 +51,8 @@ contentful:
           :preview: false
           :raise_errors: true
           :dynamic_entries: :auto
+        base_path: app_path        # Optional - Defaults to Current directory
```

Comment:
Only CDA options are symbols, though I just noticed that my specs use `base_path` as a symbol and it should be a string, fixing that